kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -413,33 +413,39 @@ public int getInitialPartitionCount()
     return initialPartitionCount;
   }
 
-  public void setClusters(String clusters)
-  {
-    this.clusters = clusters.split(";");
-  }
-
   /**
    *  Same setting as bootstrap.servers property to KafkaConsumer
    *  refer to http://kafka.apache.org/documentation.html#newconsumerconfigs
-   *  To support multi cluster, you can have multiple bootstrap.servers separated by ";"
+   *  To support multi cluster, you can have multiple elements in the array
    */
-  public String getClusters()
+  public void setClusters(String[] clusters)
   {
-    return Joiner.on(';').join(clusters);
+    this.clusters = clusters;
   }
 
-  public void setTopics(String topics)
+  public String[] getClusters()
   {
-    this.topics = Iterables.toArray(Splitter.on(',').trimResults().omitEmptyStrings().split(topics), String.class);
+    return clusters;
   }
 
   /**
-   * The topics the operator consumes, separate by','
+   * The topics the operator consumes
    * Topic name can only contain ASCII alphanumerics, '.', '_' and '-'
    */
-  public String getTopics()
+  public void setTopics(String[] topics)
+  {
```

Comment:
You can avoid creation of the intermediate ArrayList and compact the array in place with something like this but
feel free to ignore if you like the original better:
{code}
int j = 0, len = topics.length;
for (int i = 0; i < len; ++i) {
    if (null == topics[i] || 0 == topics[i].length()) continue;
    if (i != j) topics[j] = topics[i];
    ++j;
}
this.topics = Arrays.copyOfRange(topics, 0, j);
{code}